jerry-core/ecma/builtin-objects/ecma-builtin-string.c

```diff
@@ -119,10 +125,19 @@ ecma_builtin_string_dispatch_call (const ecma_value_t *arguments_list_p, /**< ar
 
   ecma_value_t ret_value = ECMA_VALUE_EMPTY;
 
+  /* 1. */
   if (arguments_list_len == 0)
   {
     ret_value = ecma_make_magic_string_value (LIT_MAGIC_STRING__EMPTY);
   }
+#ifndef CONFIG_DISABLE_ES2015_SYMBOL_BUILTIN
+  /* 2.a */
+  else if (ecma_is_value_symbol(arguments_list_p[0]))
```

Comment:
One space needed before the opening parenthesis.

Reply:
Updated 🐒